### PR TITLE
Fix Quick Start Demo of mv1p

### LINKED ISSUE #258: Quick Start Demo apps/fit/fit.py error:"Key is not in the template: args.out" working on WSL ubuntu
I'm using WSL ubuntu18.04 CUDA11.3 torch1.11.0, and have installed evrything needed according to doc.
I ran "python3 apps/demo/mocap.py data/youtube/1v1p --work internet" according to quick start, but resulted in this:

```
[run] python3 apps/calibration/create_blank_camera.py 0_input/1v1p
Read shape (1080, 1920, 3) from image 0_input/1v1p/images/0+000553+000965/000000.jpg
[run] python3 apps/fit/fit.py --cfg_model config/model/smpl.yml --cfg_data config/data/multivideo.yml --cfg_exp config/fit/1v1p.yml --opt_data "args.path" "0_input/1v1p" "args.out" "0_input/1v1p/output-smpl-3d" "args.camera" "0_input/1v1p" --opt_exp "args.stages.joints.loss.k2d.weight" "100." "args.stages.joints.loss.k2d.args.norm" "gm"
Key is not in the template: args.out
[Info] Load 1 folders: 0+000553+000965...
[Info] Load [0, -1, 1] frames
-> [Loading config/data/multivideo.yml]:  16.8ms
-> [Loading config/model/smpl.yml]:  48.1s
Loading 0+000553+000965:   0%|                                                                                                              | 0/412 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/mnt/f/ubuntu1804/EasyMocap/apps/fit/fit.py", line 38, in <module>
    fitter.fit(body_model, dataset)
  File "/home/mm1804/EasyMocap/easymocap/multistage/base.py", line 294, in fit
    for data in dataloader:
  File "/home/mm1804/anaconda3/envs/easymocap/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 530, in __next__
    data = self._next_data()
  File "/home/mm1804/anaconda3/envs/easymocap/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 570, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/mm1804/anaconda3/envs/easymocap/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/mm1804/anaconda3/envs/easymocap/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/mm1804/EasyMocap/easymocap/datasets/base.py", line 550, in __getitem__
    data = super().__getitem__(info['index'])
  File "/home/mm1804/EasyMocap/easymocap/datasets/base.py", line 409, in __getitem__
    data = super().__getitem__(index)
  File "/home/mm1804/EasyMocap/easymocap/datasets/base.py", line 301, in __getitem__
    if not os.path.exists(annname) and not self.annot_args.read:
  File "/home/mm1804/EasyMocap/easymocap/config/yacs.py", line 134, in __getattr__
    raise AttributeError(name)
AttributeError: read
Traceback (most recent call last):
  File "/mnt/f/ubuntu1804/EasyMocap/apps/demo/mocap.py", line 394, in <module>
    workflow(args.work, args)
  File "/mnt/f/ubuntu1804/EasyMocap/apps/demo/mocap.py", line 340, in workflow
    append_mocap_flags(path, output, cfg_data, cfg_model, cfg_exp, workflow.fit, args)
  File "/mnt/f/ubuntu1804/EasyMocap/apps/demo/mocap.py", line 277, in append_mocap_flags
    filenames = os.listdir(outdir)
FileNotFoundError: [Errno 2] No such file or directory: '0_input/1v1p/output-smpl-3d/smplmesh'
```

I have worked on it for a very very long time but not having a clue now.
Any advice will be appreciated!

### PR DESCRIPTION
**Pull request by xk-huang**
## tl;dr: 

Fix lightstage mv1p demo from https://chingswy.github.io/easymocap-public-doc/quickstart/quickstart.html, and other minor bugs and distracted prints.

```
python3 apps/demo/mocap.py ${data} --work lightstage-dense-smplh --subs_vis 01 --ranges 0 800 1
```

when `${data}` is relative path, the workflow fails.

## Related issue: 

Fix #258.

## Cause

`triangulation1p.py` fails.

By default the `camera` is `'none'`, then `kwargs['camera']=kwargs['path']`.
Due to data loading error from `easymocap/datasets/base.py`. https://github.com/zju3dv/EasyMocap/blob/7a80646902420692dcb2376abbd23ec5d0138d6d/easymocap/datasets/base.py#L509-L510

When the `kwargs['path']` is relative path, e.g. `./data/zju_mocap/302/`, the `camera` at https://github.com/zju3dv/EasyMocap/blob/7a80646902420692dcb2376abbd23ec5d0138d6d/easymocap/datasets/base.py#L233-L234 turn into `./data/zju_mocap/302/data/zju_mocap/302/`. Thus the camera information is not loaded, causing `keypoint2d_unproj` at https://github.com/zju3dv/EasyMocap/blob/7a80646902420692dcb2376abbd23ec5d0138d6d/easymocap/datasets/base.py#L466-L467
missing in dataloader.

## Misc

Other fixes includes, remove redundant print, check image iteration boundary, and openpose data directory with relative path.

### CODE REVIEW

**Comment by xk-huang** [comment]
Thank you so much for providing this awesome project! @chingswy
